docs/development/developing/java/adding_metrics.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Metrics Backends
+sidebar_position: 2
+---
+
+To integrate additional metrics backend into the OpenLineage client, implement the `MeterRegistryFactory` interface and ensure it is utilized by the `MicrometerProvider`'s `getMetricsBuilders` method.
```

Comment:
Where shall a user implement this? Within OpenLineage package, within Spark job or should one add additional jar with metrics backend?

Reply:
User should not implement this, contributor of OpenLineage might. I did not envision adding custom backends, rather the ones that are in https://docs.micrometer.io/micrometer/reference/implementations.html